src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw.module.ts

```diff
@@ -192,6 +192,7 @@ const routes: Routes = [
   },
   {
     path: 'multisite',
+    data: { breadcrumbs: 'Multi-site' },
```

Comment:
```suggestion
    data: { breadcrumbs: 'Multi-Site' },
```

Reply:
Thanks for the review, I just want to confirm , that do you want me to change the nav menu as well since I just copied the capitalisation from nav menu:

![image](https://github.com/ceph/ceph/assets/25664409/19bdb90d-7808-40bf-9462-08595af69573)

Thanks!

Reply:
good question. @aaSharma14 do you remember how multi-site should look like. Was it `Multi-site` or `Multi-Site`? when I looked at the docs I am seeing multiple patterns of both and I remember Rivka from doc confirmed something to us right?

Reply:
@nizamial09 , According to Rivika both `Multi-site` and `Multi-Site` are correct as long as they are hyphenated. SO we can go with either of them.